pkg/http/handlers.go

```diff
@@ -0,0 +1,23 @@
+package http
+
+import (
+	nhttp "net/http"
+)
+
+// HealthHandler is the health handler type for gobmp
+type HealthHandler struct{}
+
+// ServeHTTP is the HealthHandler interface implementation
+func (hh *HealthHandler) ServeHTTP(w nhttp.ResponseWriter, r *nhttp.Request) {}
+
+// LoadDefaultRoutes loads the default routes to be used by the HTTP server
+func LoadDefaultRoutes() *[]Route {
+	routes := make([]Route, 1)
```

Comment:
This could be simplified to the following literal: 

```
routes := [1]Route{
        {
		Path:    "/health",
		Method:  "GET",
		Handler: &HealthHandler{},
	}
}
```